src/services/http.ts

```diff
@@ -46,6 +46,14 @@ const post = <TResponse>(
 ): Promise<TResponse> =>
   makeRequest(endpoint, 'post', JSON.stringify(body), setContentTypeJSON(headers));
 
+const patch = <TResponse>(
+  endpoint: string,
+  body: Object = '',
+  headers = new Headers(),
+): Promise<TResponse> =>
+  makeRequest(endpoint, 'PATCH', JSON.stringify(body), setContentTypeJSON(headers));
```

Comment:
Why is it "PATCH" in all caps here, but "post" in lowercase above?  I think this is important, and was hard to figure out.  So it deserves a comment here.  If nothing else, it might prevent someone n the future from "cleaning it up" and breaking things mysteriously.